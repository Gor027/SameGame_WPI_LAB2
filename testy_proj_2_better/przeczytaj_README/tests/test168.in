rN rNr  NN rrNr rN rr NNrrrrNr NN  rr
rN  NNNNrNr rrN NNN N rN
 r   r NNN r rrrN rNr r NrNrr N  NN rN
 rN N r r  N    rNrr 
 rNr Nr r    NNNrNrrrr r  Nrr   rN r    NNN r rrrNNNNrNrrNNrNrr N   NN rrr N rN
NNNr NNNrN   NN rN N N r rrrNNr rNrN rr N NrNrrNNrrr Nrr rNNNNN 
rN rN  
rrrN NNr NNNNN
N N Nr rr rNrNrr  r  Nr N rNN rr
Nr NNrNrr r   r r   rrrrN N Nr
rN N     NNNNNNrNN N rN rrrr N   rrrr   NNrrr N rrr rNNNNNNrrrr rrr
r r Nr rrrrrrrNrNrNr NN      NN rNNNNN rN NN 
NNrrN  rNN  NNNNrN  rrr N N rrN N  rrNr Nr rNrNN rN rNNNr Nrrrr N    rNNrrN
r
r  NNrr N  rrr r NNNrN   r NN r N r  r Nr rr r N rr N  N N 
rN  r    NNNN Nr N NN
Nr rN Nr NNNN rr r   rNrrrNrN   r  NNrNr  Nr  Nr r 
