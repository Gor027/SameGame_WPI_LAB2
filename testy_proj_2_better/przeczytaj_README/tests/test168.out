rN
rN            r
Nr  N   N    NN  N
rr rNr  rN rrrr rN
rr rNN NNN rrrNrNN r rN   r N
NN rNrNNrNrNNNNNNN NrNr   Nrr
NNNrNNrNrNNNrNrrrNNrNNr r Nrr   N  r
rrNNNNrrrNrNNNrrrNrrrrrNNrrrNrN r  rrN  N Nr     N
rrNNNNNrrNrNrNNrrrrNrrrNNrrrNNr rNNNrr  N Nr  rr rN    r
NNrNNrNrrrNrNNNNrNrNrNNrNrNNNrr rNrrrN  N rNr rN rr NrNr NNrNrr
rNNrNrNrrNrrrrNrNNrrrrNrrrrrrNN NNrNNNrrNNNrNrNNrrr rNrNrrNNNNNN
rNrNrrrrNNNNNNNrNNNNrNrrNrNNNNNNNrNrrrrNNNrrrrNNNrNNrNNNNNNrrrrr   N rrr
rrrrNrNrNNNNNrrrrrNNNrNrrrNrNrrrrNrNNrNrrrNrNrNrrrrrNNNNrNNrrrrNrrNrNNrrNNrN
